Leetcode/1897/gukin-han.java

```diff
@@ -0,0 +1,18 @@
+class Solution {
+    public boolean makeEqual(String[] words) {
+        HashMap<Character, Integer> map = new HashMap<>();
+
+        for (String word: words) {
+            for (char c: word.toCharArray()) {
+                map.put(c, map.getOrDefault(c, 0) + 1);
+            }
+        }
+
+        int n = words.length;
+        for (Character ch: map.keySet()) {
+            if (map.get(ch) % n != 0) return false;
+        }
```

Comment:
```suggestion 
       for (Integer count: map.values()) {
            if (count % n != 0) return false;
        }
```

`values()` 메서드를 사용하면 조금 더 빨라질 수 있을 것 같습니다!

Reply:
덕분에 오늘 문제에서는 values()를 적용해보았습니다